README.md

```diff
@@ -78,10 +78,7 @@ class PageController extends Controller
     {
         $page = Page::findBySlug($slug);
 
-        if (!$page)
-        {
-            abort(404, 'Please go back to our <a href="'.url('').'">homepage</a>.');
-        }
+        abort_if(!$page, 404, 'Please go back to our <a href="'.url('').'">homepage</a>.');
```

Comment:
If I just got the versions right, the current dependency will not for sure lead to Laravel >= 5.3.

So `abort_if` may not be defined.

Reply:
Backpack [officialy requires Laravel 5.5+](https://laravel-backpack.readme.io/docs/getting-started#section-requirements) so that shouldn't be a thing.